README.md

```diff
@@ -245,7 +247,9 @@ when initializing your instance of the SplunkRum API:
   Set/change the default polling interval for slow/frozen render detection.
   Default is 1000ms. Value must be positive. 
 - `disableSlowRenderingDetection()` :
-  Disable the detection of slow frame renders (default is enabled). 
+  Disable the detection of slow frame renders (default is enabled).
+- `enableDiskBuffering()` : 
```

Comment:
@breedx-splk I've changed some of these names in #255 to match the other properties, please take a look at that

Reply:
Cool, I think I've got it updated to match.